### ISSUE TITLE
API GET /voters does not fetch aged voters

### ISSUE THREAD

**Issue by liberspirita** [LiskHQ/lisk#2429]
### Expected behavior
Getting all voters of a delegate, whenever the vote occured

### Actual behavior
Partial results : old voters do not appear

### Steps to reproduce
For example : 15221539038729566286L voted on many delegates in 2016. If you apply API getVoters on those delegates, you wont have 15221539038729566286L  in results

### Which version(s) does this affect? (Environment, OS, etc...)
core 1

**Comment by nazarhussain** [comment]
@liberspirita If I go to the the account I can't see if they voted for you or not. 

https://explorer.lisk.io/address/15221539038729566286L

This account voted to 67 delegates but none is to `liberspirita`. If you go to transaction details for that account, it never send any vote transaction in 2016.

**Comment by liberspirita** [comment]
There is a misunderstanding, delegate liberspirita is not all all in my example given.
look this example : 
do a /GET voters on 15458001843048965724L
Do you find 15221539038729566286L ?
I think you won't .... but this address voted on 15221539038729566286L long time ago... and did not devoted since

**Comment by liberspirita** [comment]
I think it is related to issue #2330 
sorting is the issue, but also something else ... Because it is obvious that when fetching all offsets, at the end the older voters are missing

**Comment by Lisk-Star-Lottery** [comment]
> There is a misunderstanding, delegate liberspirita is not atall in my example given.
> look this example :
> Use API /GET voters on 15458001843048965724L
> Do you find 15221539038729566286L ?
> I think you won't .... but this address voted on 15221539038729566286L long time ago... and did not devoted since

i confirm: 
**voter "15221539038729566286L"** can not be find as voter on **delegate "15458001843048965724L" aka "leo"** by using https://explorer.lisk.io/delegate/15458001843048965724L

**Comment by ducktank1990** [comment]
i confirm:
**voter "15221539038729566286L"** can not be find as voter on **delegate "15458001843048965724L" aka "leo"** by using https://explorer.lisk.io/delegate/15458001843048965724L

**Comment by nazarhussain** [comment]
Hi @liberspirita I found the vote you are talking about.

```
curl -X GET  "public.lisknode.net:8000/api/voters?address=15458001843048965724L&limit=100&offset=200" | jq -c '.data.voters[] | select(.address == "15221539038729566286L" )'
```

But yes if due to bad sorting issue it does not appear in the right offset sometime. And will be fixed by #2330

**Comment by nazarhussain** [comment]
Duplicate of https://github.com/LiskHQ/lisk/issues/2330
